server/apps/main/templates/main/moderation_list.html

```diff
@@ -31,27 +33,29 @@
 <section id="moderation-table">
   <div class="row">
     <div class="col-lg-12">
-      <div class="card-header"><h4>Stories</h4></div>
-      <div class="card-body">
-        <div class="container search">
-          <form method="GET" action="{% url 'main:moderation_list' %}">
-            <div class="form-row align-items-center" style="left: 0px">
-              <div class="col-lg-10">
-                <label class="sr-only" for="inlineFormInput">Name</label>
-                <input class="form-control mb-2"
-                        id="inlineFormInput"
-                        placeholder="Search stories"
-                        type="text"
-                        name="searched"
-                        value="{{ searched }}"
-                        />
-              </div>
+      <div class="card border-light">
+        <div class="card-header"><h4>Stories</h4></div>
+        <div class="card-body">
+          <div class="container search">
+            <form method="GET" action="{% url 'main:moderation_list' %}">
+              <div class="form-row align-items-center" style="left: 0px">
+                <div class="col-lg-10">
+                  <label class="sr-only" for="inlineFormInput">Name</label>
+                  <input class="form-control mb-2"
+                          id="inlineFormInput"
+                          placeholder="Search stories"
+                          type="text"
+                          name="searched"
+                          value="{{ searched }}"
+                          />
+                </div>
 
-              <div class="col-auto">
-                <button class="btn btn-primary mb-2" type="submit">Search</button>
+                <div class="col-auto">
+                  <button class="btn btn-primary mb-2" type="submit">Search</button>
+                </div>
               </div>
-            </div>
-          </form>
+            </form>
+          </div>
         </div>
       </div>
```

Comment:
Something that becomes apparent in dark mode is that these cards have a border. It's barely visible in light mode but in dark mode it's really high contrast. But it highlights that there's no gap between the "Stories" card above and the "Review tabs" below.

It might be worth adding in some kind of gap, e.g. by adding in a top margin to the `moderation-table-container`:
```
.moderation-table-container {
  margin-top: 3%;
  margin-bottom: 3%;
} 
```